indy_common/auth.py

```diff
@@ -32,6 +32,10 @@ class Authoriser:
             {TRUSTEE: []},
         '{}_role_{}_'.format(NYM, TRUST_ANCHOR):
             {TRUSTEE: []},
+        '{}_role_<any>_<any>'.format(SCHEMA):
+            {TRUSTEE: [OWNER, ], STEWARD: [OWNER, ], TRUST_ANCHOR: [OWNER, ]},
+        '{}_role_<any>_<any>'.format(CLAIM_DEF):
+            {TRUSTEE: [OWNER, ], STEWARD: [OWNER, ], TRUST_ANCHOR: [OWNER, ]},
```

Comment:
An existing field of the transaction should be used rather than not existing (`role`). For `CLAIM_DEF` transaction it makes sense to use `ref` filed here.